src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1507,9 +1507,7 @@ PeerImp::onMessage(std::shared_ptr<protocol::TMTransaction> const& m)
             }
         }
 
-        // The maximum number of transactions to have in the job queue.
-        constexpr int max_transactions = 250;
-        if (app_.getJobQueue().getJobCount(jtTRANSACTION) > max_transactions)
+        if (app_.getJobQueue().getJobCount(jtTRANSACTION) > max_transactions_)
```

Comment:
Consider instead:

    if (app_.getJobQueueCount().getJobCount(jtTRANSACTION) > app_.config().MAX_TRANSACTIONS)

Rationale: the variable is only set in one place and isn't "per-peer" so this lets you eliminate the unnecessary member variable, and highlights the intent of the code.